### DISCUSSION TITLE
Moje spostrzeżenia

### DISCUSSION THREAD

**Discussion by xrazdwa** [mariusz-ostoja-swierczynski/tech-controllers#51] [General]
Jestem użytkownikiem listwy do sterowania ogrzewaniem podłogowym L-9R oraz modułu wifi M-9R od techa.  Na początku wielkie dzięki, że chciało Ci się podjąć wyzwanie z tą integracją. Nie mam zbyt dużego doświadczenia z Home Assistant (dopiero co zainstalowałem) ale pozwolę sobie dopisać swoje obserwacje. 
1. po dodaniu integracji pojawiły mi sie w HA termostaty moich stref co mnie zdziwiło, że przy próbie zmiany temperatury w termostacie wartość przeskakuje o 0,5 stopnia, (w emodul jest o 0,1 stopnia.)
2. nie widzę w HA wartości temperatury z bezprzewodowego czujnika temp zewnętrznej - (mam podpięty w swojej instalacji czujnik C-8 ZR - w emodul.eu wyświetlana jest jego wartość)
3. Część termostatów w mojej instalacji działa według harmonogramu, jeśli w HA kliknę w +/- aby zmienić temp. moje harmonogramy automatycznie zostają wyłącznone.
4. Nie ma informacji na temat stanu baterii w czujnikach bezprzewodowych (przydała by się)
5. fajnie byłoby mieć informacje na temat wilgotności - ale tego nawet w emodul nie ma (mimo, że moje regulatory podają taką informację.)

pozdrawiam
ps. wrazie czego chętnie mogę robić za testera.

**Comment by chappaai** [comment]
1. Zgadzam się. 0.5 wydaje się zbyt wysoką wartością kroku. 0.1 + możliwość wpisania własnej wartości było by bardziej OK. 
3. To jest wg mnie scenariusz, który był do przewidzenia. Tak samo w przypadku, jeżeli termostat działa wg harmonogramu i ręcznie, manualnie zostanie zedytowana wartość temperatury wtedy harmonogramy zostają zatrzymane. Rozwiązaniem jest przejście na harmonogramy sterowane z HA. 
4. U mnie jest 
5. Termostaty z czujnikiem wilgotności poprawnie podają wartość wilgotności.

**Reply by daroga0002** [reply]
1 zostało zmienione na main branch, pewnie za jakąś chwile bedzie nowy release, 4 i 5 u mnie dziala na L-X WiFi
